cypress-prioritization-and-triage.md

```diff
@@ -0,0 +1,123 @@
+
+Here are the criteria we use to gauge relative issue priority:
+
+| Criteria | Weight | Description |
+|:---------------------|:---------:|:-----------------------------------------------------------------------------------|
+| Scope | 1.0 | How many users does this affect? |
+| Severity | 1.0 | What does the problem prevent users from doing? Is it an edge case? A primary test flow? A wrong label or errant CSS? |
+| Visibility | 1.0 | Is this an important issue to our community? Is there a lot of discussion around it? |
+| Does Workaround Exist | 1.0 | Does a work around exist that is reasonable? |
+| Regression | 1.0 | Did this used to work? How long ago did it last work? Was this a regression within this last (current) major version? |
+| Cypress Prioirty | 1.0 | Does Cypress have a vested interest in resolving this issue? Example, does it ease support burden for our staff? Is this part of a corporate milestone or objective? |
+| Effort | 1.0 | How much effort is needed to resolve the issue? Estimates are for a single developer being assigned to the issue. |
+
+The formula for calculating the Cypress Relative Priority Score (CRPS) is the weighted sum of our priority criteria:
+
+
+$$ \text{Cypress Relative Priority Score} = {{(Scope \times 1.0) + (Severity \times 1.0) + (Visibility \times 1.0) + (WorkAround \times 1.0) + (Regression \times 1.0) + (CypressPriority \times 1.0)}} $$
+
+
+It is important to remember that just because an issue has a high CRPS, that doesn't mean it is necessarily the best way to allocate limited resources. To determine which gives us the greatest bang for the proverbial buck, we weigh CRPS versus the effort required to address the issue.
+
+$$ \text{Cypress Relative Priority to Effort Score} = {\text{Cypress Relative Priority Score} \over {Effort}} $$
+
+**Cypress Relative Priority to Effort Score** gives us a metric that better represents the effort-to-reward ratio for any proposed work.
+
+## FAQs
+
+### Why isn't anyone working on my issue?!
+
+We understand that it can be frustrating to take time to enter a ticket, only to see it sit in the backlog untouched for a long period of time. We truly wish we could take up every single issue that comes in, but given the relatively small size of our internal teams and the large and varied user base of Cypress it just isn't possible to solve every issue. Our prioritization process helps us float the most important issues to the top based on the impact vs effort ratio of any given issue. It is important to remember that, just because an issue is not being actively worked on, does not mean we are ignoring the issue; it means that we have other issues we feel are more impactful to the user base and are prioritizing those issues first.
```

Comment:
```suggestion
We understand that it can be frustrating to take time to submit an issue, only to see it sit in the backlog untouched for a long period of time. We truly wish we could take up every single issue that comes in, but given the relatively small size of our internal teams and the large and varied user base of Cypress it just isn't possible to solve every issue. Our prioritization process helps us float the most important issues to the top based on the impact vs effort ratio of any given issue. It is important to remember that, just because an issue is not being actively worked on, does not mean we are ignoring the issue; it means that we have other issues we feel are more impactful to the user base and are prioritizing those issues first.
```

Reply:
addressed